internal/pkg/externalplugins/lgtm/lgtm.go

```diff
@@ -50,16 +51,15 @@ func HelpProvider(epa *externalplugins.ConfigAgent) func(
 			}
 		}
 		pluginHelp := &pluginhelp.PluginHelp{
-			Description: "The ti-community-lgtm plugin manages the application and " +
-				"removal of the 'status/LGT{number}' (Looks Good To Me) label which is typically used to gate merging.",
-			Config: configInfo,
+			Description: "The ti-community-lgtm plugin manages the 'status/LGT{number}' (Looks Good To Me) label.",
+			Config:      configInfo,
 		}
 
 		pluginHelp.AddCommand(pluginhelp.Command{
-			Usage:       "/lgtm [cancel] or GitHub Review action",
-			Description: "Adds or removes the 'status/LGT{number}' label which is typically used to gate merging.",
+			Usage:       "/lgtm [cancel]",
+			Description: "Add or remove the 'status/LGT{number}' label.",
 			Featured:    true,
-			WhoCanUse:   "Collaborators on the repository. '/lgtm cancel' can be used additionally by the PR author.",
```

Comment:
ditto

Please revert `'/lgtm cancel' can be used additionally by the PR author.`.

Reply:
I've removed this feature.

Reply:
Please revert this feature. 
I think this is a useful feature, if you think people will never use this feature, please create an issue or a discussion and we'll talk together to decide whether to remove it.

Reply:
@andylokandy  ping~